src/title/title.sss

```diff
@@ -1,3 +1,4 @@
+.class_title,
```

Comment:
We do not need extra class if the content is the same (also CSS classes is following BEM)

Reply:
oh, forgot to delete it when removing the group names from the page